docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -127,22 +127,9 @@ export function Dino() {
 }
 ```
 
-```jsx
-// styled-components
-
-export function Dino() {
-  return (
-    <StaticImage
-      src="trex.png"
-      css={`
-        border: 4px green dashed;
-      `}
-    />
-  )
-}
-```
+Unfortunately the [`css` prop from styled-components](https://styled-components.com/docs/api#css-prop) turns the code into a `styled` function under the hood and as explained above `StaticImage` doesn't support that syntax.
```

Comment:
It feels like there's a second sentence missing from this paragraph. Something about what the implication is for the user. Like, "If you're using styled-components, you should do X instead." Or can you just not use styled-components with it?

Reply:
Yeah, you can't use styled-components with this. That's pretty much the implication.